extensions/ql-vscode/src/run-queries.ts

```diff
@@ -632,13 +631,7 @@ export async function compileAndRunQueryAgainstDatabase(
   const metadata = await tryGetQueryMetadata(cliServer, qlProgram.queryPath);
 
   let availableMlModels: cli.MlModelInfo[] = [];
-  // The `capabilities.untrustedWorkspaces.restrictedConfigurations` entry in package.json doesn't
-  // work with hidden settings, so we manually check that the workspace is trusted before looking at
-  // whether the `shouldInsecurelyLoadMlModelsFromPacks` setting is enabled.
-  if (workspace.isTrusted &&
```

Comment:
Should we still prevent this setting in untrusted workspaces?

Reply:
I'm not sure I understand your question — this setting is no longer used.

Reply:
Sorry...that was unclear. I just meant is it safe to use ml models in an untrusted workspace? The workspace may contain an untrusted model, which as I understand it, could run untrusted code.

I know there are SHAs that ship with the CLI to restrict which models are allowed to run. Does that mean there is no danger of running an untrusted model?